general-concepts/git/text.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<guide self="general-concepts/git/">
+<chapter>
+<title>Git for Gentoo Developers - General concepts</title>
+
+<section>
+<title>Certificate of Origin</title>
+<body>
+
+<p>
+Per Gentoo's <uri link="https://www.gentoo.org/glep/glep-0076.html">GLEP 76
+(Copyright Policy GLEP)</uri>, you must sign-off all your commits to any
+Gentoo-hosted repository with accordance to the
+<uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
+GLEP policy</uri>. 
+</p>
+
+<p>
+When committing work authored by someone else, e.g. a bugzilla patch, or GitHub
+pull request, a sign-off from the original author is always strongly
+recommended to have to indicate the author acknowledges Gentoo's copyright
+policy. However, it is not mandatory for every case. Please refer to the
+example list below when determining whether a sign-off from the original author
+is, or is not required. The list below serves as a general guideline.
+</p>
+
+<p>
+If the author has properly signed off their commits, per 
+<uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
+GLEP 76</uri> you are <b>prohibited</b> in making your own changes to those
+commits, with author's name. Please do a separate, follow-up "fix" commit. The
+original contribution should be worked to be in a good state before merging,
+where any follow-up fixes are purely cosmetic, or minor by nature.
+</p>
```

Comment:
That's not a policy we have.

It makes no sense to clutter the history by cosmetic follow-up commits. So, either ask the author to fix their commit, or follow a procedure resembling the one outlined in the [Linux commit policy](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/Documentation/process/submitting-patches.rst?h=v4.18#n466), i.e. add a short notice _below_ the author's signoff.

(Also, sometimes modifying the original commit is inevitable, e.g. when it needs to be rebased.)

Reply:
Maybe not a policy, but definitely a widely adopted habit. 

So again the GLEP has what seems like a clear and strict condition, which is apparently up to interpretation and can be passed. At least I'm writing what you can do, since your alternative way is not known to people. And I personally like the GLEP-66 way of applying "atomic commits per changes". So I disagree with you here, until GLEP-76 is clarified one way or another.

Reply:
Then drop this paragraph altogether please. The devmanual make recommendations about which there is disagreement.

Reply:
Done and done.